db/migrate/20201112214758_create_contact_attempts.rb

```diff
@@ -0,0 +1,45 @@
+class CreateContactAttempts < ActiveRecord::Migration[6.0]
```

Comment:
Since we had to go with the first iteration of the contact attempts feature for the 1.17.0 release rather than having these changes replace it, this migration should now also drop the `contact_attempts` column on the `patients` table.